app/models/route_set.rb

```diff
@@ -81,6 +81,24 @@ def register!
     commit_routes
   end
 
+  def delete!
+    return unless any_routes?
+
+    paths =
+      routes.map { |route| route.fetch(:path) } +
+      redirects.map { |redirect| redirect.fetch(:path) }
+
+    paths.each do |path|
+      begin
+        Rails.application.router_api.delete_route(path)
```

Comment:
Perhaps not related to this PR, but would be nice to get `Rails.application.router_api` into a method to avoid having to write `Rails.application` everywhere.

Reply:
As you've commented about it, I've added separate commit making this change. I can always move it to a separate Pull Request if you'd like.